Refit/IO/PooledMemoryStream.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Buffers;
+using System.IO;
+
+namespace Refit.IO
+{
+    /// <summary>
+    /// An in-memory <see cref="Stream"/> that uses memory buffers rented from a shared pool
+    /// </summary>
+    public sealed class PooledMemoryStream : Stream
```

Comment:
Looks like `PooledMemoryStream` should probably be `internal`

Reply:
Yup, absolutely.
I made it `public` while just writing down a draft and forgot to change that 👍